shared/enclave/server/wallets/keepkey.ts

```diff
@@ -0,0 +1,21 @@
+import { WalletLib } from 'shared/enclave/types';
+
+const KeepKey: WalletLib = {
```

Comment:
Instead of the repetition of implementing a wallet lib here, we should make a base class that has all of the methods throw the unimplemented error

Reply:
I initially decided against classes in favor of singletons since there aren't multiple instances of any of these, they're moreso handlers than instances. However I could still make them classes and just export an instance of them, so I guess that makes more sense.

The util function makes sense though, I'll try to work that out.